### ISSUE TITLE
provR and prov.annotate.on

### ISSUE THREAD

**Issue by fong22e** [End-to-end-provenance/RDataTracker#491]
In basicTest, there is a line that calls `prov.annotate.on`, which makes the test fail while using provR as that function is not part of provR's exported functions. Should I remove that line of code from the test or should there be a dummy function in provR for it? I was hoping to use the same test file for both rdt and provR tests. Is that not a good idea?

**Comment by blernermhc** [comment]
I agree that we should use the same tests for both tools as much as possible.  I would suggest removing the call to prov.annotate.on.  Then, we should create 1 test that contains RDT-only code that is only run against RDT.

**Comment by fong22e** [comment]
Sounds good. Currently, it is just that call to prov.annotate.on that is causing the problem. There are also other bugs in other tests that I have yet to debug.

**Comment by fong22e** [comment]
Is this the correct ddg for a test that contains only the line `prov.annotate.on("f")`?
![tmp](https://user-images.githubusercontent.com/20979904/47318442-1420fc80-d61a-11e8-875c-0c5eed3af644.png)

**Comment by blernermhc** [comment]
That's a good question.  We generally don't add nodes for calls to functions inside RDT, thinking that those were added via annotation.  But, perhaps if the user wrote the call, we should.  So, perhaps we should add nodes if the call is one of the functions exported from the API file.

**Comment by fong22e** [comment]
May I ask, how much of a priority is this? This does not seem like an easy thing to do...

**Comment by blernermhc** [comment]
Do scripts with a single line of regular non-rdt R code build the ddg you expect?

**Comment by fong22e** [comment]
Yes, it works fine.

**Comment by blernermhc** [comment]
Ok, I don't think we need to change anything for the CRAN release.  I have left it here as a low priority issue, and not tied to the CRAN milestone.  We can return to this later and decide what we want.

**Comment by fong22e** [comment]
Sounds good!
